packages/vanilla/src/sass/components/banner/_mixin.scss

```diff
@@ -1,53 +1,42 @@
-//@use '../../base/colors' as *;
-// @use '../button' as *;
+@use '../../layout/container' as *;
 
 @mixin banner-info($bg-color) {
   background-color: $bg-color;
-  padding-top: 16px;
-  padding-bottom: 16px;
+  padding-top: var(--cbp-space-4x);
+  padding-bottom: var(--cbp-space-4x);
 
   & > div {
     @content;
-    margin: 0 16px;
+    @extend .cbp-fluid-container;
 
     & > div:first-child {
       display: flex;
       flex-shrink: 0;
     }
-
-    @media (min-width: 600px) and (max-width: 1439px) {
-      margin: 0 20px;
-    }
-
-    @media (min-width: 1440px) {
-      margin: 0 28px;
-    }
   }
 }
 
 @mixin banner-text($text-color) {
   color: $text-color;
   font-size: 14px;
```

Comment:
Replace with css variable

Reply:
I just removed this and the weight, which are identical to body text, which this should inherit since it's just a `p` tag.